tests/login.test.ts

```diff
@@ -0,0 +1,61 @@
+import { test, expect } from '@playwright/test';
+import { HomePage } from '../pages/home.page';
+import { LoginPage } from '../pages/login.page';
+import { MyAccountPage } from '../pages/myAccount.page';
+import { HeaderComponent } from '../components/header.components';
+import { userCredentials } from '../test-data/users.data';
+
+test.describe('Login tests', () => {
+    let loginPage: any;
+
+    test.beforeEach(async ({ page }) => {
+        const homePage = new HomePage(page);
```

Comment:
We could add fixtures for our tests please add file. 

In file 'pagesTypes.ts' we will aggregate all types for pageObjects

```
import { LoginPage } from '@pwPages/login.page';

export type LoginPageType = ReturnType<typeof LoginPage>;
```

In file 'pages.ts' we will aggregate all pages 

```
import { LoginPage } from '@pwPages/login.page';

export {
  LoginPage,
};
```

playwrightTest.ts

```
import { test as base } from '@playwright/test';
import * as PagesTypes from '@pwPages/pagesTypes';
import * as pages from '@pwPages/pages';

type Pages = {
  loginPage: PagesTypes.LoginPageType;
  // add other page objects
};

export const test = base.extend<Pages>({
  async loginPage({ page }, use) {
    await use(pages.LoginPage(page));
  },
});

export { expect } from '@playwright/test';
```

Reply:
Done